### DISCUSSION TITLE
How to increase/decrease font-size, add HTML and angel of arcLabel in Nivo/Pie?

### DISCUSSION THREAD

**Discussion by Vinay-logiciel** [plouc/nivo#2124] [Q&A]
Hi there,

1. I wanted to **increase or decrease the font size of arcLabel** in @nivo/pie
2. How can I **rotate my text on arcLable** at a specific angle.
3. I want to customize the label on the Pie-slice. I **wanted to add some HTML code** like want to add a button or something else. How can I do that? Can you please show me a quick and simpler way?

**Comment by Ashutosh-Maurya-87** [comment]
@tony  @martingordon  @andresgutgon  @recurser  Please Reply facing same issue.

**Comment by andresgutgon** [comment]
Hi, I'm not a Nivo maintainer. I'm just using it so I can't talk on behave of Nivo's maintainers. **BUT** I want to say this is not the proper place to ask questions on how to use Nivo. Issues on the repo are usually for bugs or improvement suggestions. 

For questions Nivo have a Discord channel. Also Nivo documentation is great. Did you read it? And last but not least is the source code. Read it, understand what's doing and maybe that can give you ideas on how to solve your problem

@Ashutosh-Maurya-87 don't ping me if I'm not related at all with the issue please 🙏

**Comment by plouc** [comment]
Hi,

1. You can use the theme.
2. You cannot.
3. The chart is in SVG.

**Reply by shter2499** [reply]
how do i change only link text on pie, now i'm using labels but it change both of number and link text
![image](https://github.com/plouc/nivo/assets/75008160/b6c36dcb-40d4-4682-b3b4-2c750b0bf47f)
![image](https://github.com/plouc/nivo/assets/75008160/aa7312ca-0a1b-4701-b5d6-d31d9c30e9ee)
